src/backend/cli.ts

```diff
@@ -0,0 +1,91 @@
+import fs from 'fs';
+import { promisify } from 'util';
+import yargs from 'yargs';
+import { createApp } from './app';
+import { fetchOpenDataIndex } from './dataExports/openDataIndex';
+import { fetchTotalResponses } from './dataExports/totalResponses';
+import { fetchCityLevelGeneralResults } from './dataExports/cityLevelGeneralResults';
+// NOTE: To be added in another PR
+// import { fetchCityLevelWeeklyGeneralResults } from './dataExports/cityLevelWeeklyGeneralResults';
+import { fetchDailyTotals } from './dataExports/dailyTotals';
+
+const writeFile = promisify(fs.writeFile);
```

Comment:
There's also `fs.writeFileSync`, which may save one or two lines of having to promisifying `writeFile`, or is there a particular reason for using writeFile specifically here?

Then again, I have no problem with using `writeFile` neither :man_shrugging:  : D

Reply:
Good question, there's 2 personal philosophy items in play here:

1. I try to avoid using any of the `***Sync` functions of the standard library. Javascript is fundamentally asynchronous based on an underlying event loop, and I feel like these sync-variants fight against the very nature of the platform.
2. The actual call happens in `async` function, so there's no loss of coherency. Syntactically it _looks_ almost the same as a synchronized call and Typescript makes sure I don't forget the `await`. Praise to `async` functions that can help us to do the right thing in an ergonomic way.

As a personal opinion, the Node.js API is in a bad place which is hard to recover from when it comes to these sync/async functions. `promisify` is a useful addition, but the ergonomics would be so much better if we had native access to already promisifed functions. Here's hoping that will be added someday 🤞

Reply:
Thanks for pouring part of your heart out on this one 😇  Let's keep the hope high in this case : D